cmd/skipper/filterFlags.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"strings"
+)
+
+const filterPluginUsage = "set a custom filter plugins to load, a comma separated list of name and arguments"
+
+type filterFlags struct {
+	values [][]string
+}
+
+func (f *filterFlags) String() string {
+	var ret []string
+	for _, val := range f.values {
+		ret = append(ret, strings.Join(val, ","))
+	}
+	return strings.Join(ret, " ")
+}
+
+func (f *filterFlags) Set(value string) error {
```

Comment:
is the logic exactly the same for filterFlags, predicateFlags and dataclientFlags? If yes, is it possible to create only type for them, s.g. like pluginFlags, and use it in each case?